x/concentrated-liquidity/genesis_test.go

```diff
@@ -188,14 +211,22 @@ func (s *KeeperTestSuite) TestInitGenesis() {
 					},
 					positionData: []genesis.PositionData{
 						{
-							LockId:         1,
-							Position:       &testPositionModel,
-							FeeAccumRecord: testFeeAccumRecord,
+							LockId:            1,
+							Position:          &testPositionModel,
+							FeeAccumRecord:    testFeeAccumRecord,
+							UptimeAccumRecord: testUptimeAccumRecord,
 						},
 						{
 							LockId:         0,
 							Position:       withPositionId(testPositionModel, 2),
 							FeeAccumRecord: testFeeAccumRecord,
+							UptimeAccumRecord: []accum.Record{
+								accumRecordWithDefinedValues(accumRecord, sdk.NewDec(10000), sdk.NewInt(100), sdk.NewInt(50)),
+								accumRecordWithDefinedValues(accumRecord, sdk.NewDec(1000), sdk.NewInt(100), sdk.NewInt(50)),
+								accumRecordWithDefinedValues(accumRecord, sdk.NewDec(100), sdk.NewInt(100), sdk.NewInt(50)),
+								accumRecordWithDefinedValues(accumRecord, sdk.NewDec(10), sdk.NewInt(100), sdk.NewInt(50)),
+								accumRecordWithDefinedValues(accumRecord, sdk.NewDec(1), sdk.NewInt(100), sdk.NewInt(50)),
+							},
```

Comment:
I see you added UptimeAccumRecords to both the import and export test cases, but where are we checking that it was properly imported or exported?

Reply:
yup 

export: https://github.com/osmosis-labs/osmosis/pull/5076/files#diff-98c98ae5496ec51279e773847b7deb617103edd50a19b6ffa78a7f4889f8307eR813

import: https://github.com/osmosis-labs/osmosis/pull/5076/files#diff-98c98ae5496ec51279e773847b7deb617103edd50a19b6ffa78a7f4889f8307eR564